### PR TITLE
[completer] Fix bug which causes integer divide by zero panicking when input buffer is empty

### PR DESCRIPTION
**Pull request by antekresic**
Fixes issue #53

### CODE REVIEW

File: complete.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func (o *opCompleter) OnComplete() {
 	buf := o.op.buf
 	rs := buf.Runes()
 
-	if o.IsInCompleteMode() && runes.Equal(rs, o.candidateSource) {
+	if o.IsInCompleteMode() && len(rs) > 0 && runes.Equal(rs, o.candidateSource) {
```
Comment (chzyer) [inline complete.go]: Use `runes.Equal(rs, o.candidateSource)` here is want to indicate whether user press twice `<tab>`.
`o.candidateSource` will be set to nil when `o.ExitCompleteSelectMode()` is called.
The problem is `runes.Equal([], nil)` is true.
So I suggests that change `len(rs)` to `o.candidateSource != nil`

**Comment by chzyer** [comment]
Just to mention, the user who submit the commit is not a valid github user :)
Would you want to set it to your github account ?

**Comment by antekresic** [comment]
Updated the PR and, hopefully, fixed the user. 

Thanks for the heads up!

**Comment by chzyer** [comment]
👍

**Pull request merged**